pkg/executor/nginx/nginx.go

```diff
@@ -274,8 +274,9 @@ func nginxLocationsMapToString(m nginxLocations, documentRoot string, path strin
 		singleLocation := fmt.Sprintf("%slocation %s%s {\n", indentN1, path, key)
 		singleLocation = fmt.Sprintf("%s%sroot %s;\n", singleLocation, indentN2, documentRoot)
 
-		gZipUse := strings.TrimSpace(value.Gzip.Use)
-		if gZipUse == "on" || gZipUse == "off" {
+		gZipUseStatic := strings.TrimSpace(value.Gzip.UseStatic)
+
+		if gZipUseStatic == "on" || gZipUseStatic == "off" {
 			singleLocation = getGzipConfAsString(value.Gzip, singleLocation, indentN2)
```

Comment:
Klarer du å ha mindre konfigurasjon?

Det bør holde med "gzip": true. 

Vi skal gzippe statiske JS-filer, HTML og CSS. Det holder.

Reply:
en av oss har misforstått.. det pleier å være meg. Vi skal vel ikke gzippe noe? Det er det jeg har tatt bort

Reply:
Dårlig formulert.

Slik jeg leser koden nå så er det endel konfigurasjon av gzip_static. Trenger vi det? Kan vi ikke bare si at vi slår det på og har fornufte settings på resten?

Reply:
ah, sånn å forstå. Ja, det er nok bedre. Skal se på det

Reply:
Hva synes du nå @nixx ?

Reply:
Veldig bra. Da må vi bare sørge for å få testet det godt gjennom alle mellomtjenere...